src/plugins/search/search.js

```diff
@@ -278,9 +277,9 @@ export function init(config, vm) {
 
   const isExpired = localStorage.getItem(expireKey) < Date.now();
 
-  INDEXS = JSON.parse(localStorage.getItem(indexKey));
+  INDEXS = isExpired ? null : JSON.parse(localStorage.getItem(indexKey));
 
-  if (isExpired) {
+  if (!INDEXS) {
```

Comment:
When manually delete `docsify.search.index` and keep `docsify.search.expires`, this plugin can still operate normally